config/manifests/inferencemodel.yaml

```diff
@@ -8,27 +8,5 @@ spec:
   poolRef:
     name: vllm-llama3-8b-instruct
   targetModels:
-  - name: food-review
+  - name: food-review-1
     weight: 100
-
----
-apiVersion: inference.networking.x-k8s.io/v1alpha2
```

Comment:
those base models are created intentionally, but we can certainly use better names for those resoruces @kfswain

Reply:
Add base models back to the file.